src/core/mac/mac.hpp

```diff
@@ -683,6 +683,8 @@ class Mac : public InstanceLocator
 
     static const char *OperationToString(Operation aOperation);
 
+    SubMac mSubMac;
```

Comment:
I guess you changed this for testing only. Right? (and will remove it before merge?)
I expect this change can cause a RAM size increase (by ~8 or so ). But it's interesting that we see code size change as well...

Reply:
Yes, I will revert this change, which just shows this works.